src/ScraperProxy/scraper.cpp

```diff
@@ -780,52 +745,14 @@ bool ProcessProjectRacFileByCPID(const std::string& project, const fs::path& fil
             fs::remove(file);
 
         return false;
-    }
+    }*/
 
     _log(INFO, "ProcessProjectRacFileByCPID", "Finished processing " + file.string());
     _log(INFO, "ProcessProjectRacFileByCPID", "Started processing CPID rac data and stripping");
 
-    // Strip the rac xml data to contain only why we need to be compressed from vector
-    // We Need:
-    // total_credit, expavg_time, expavg_credit, teamid, cpid
-    // We Don't need:
-    // id, country, name, url
-    // In reality we DO NOT need total_credit till TCD
-    // In reality we DO NOT need expavg_time but this will make nn compatible
-    // I've also opted to save 1 byte a line by not doing newlines since this a scraper
-
-    stringbuilder xmlout;
-
-    xmlout.nlappend("<users>");
-
-    for (auto const& vv : vXML)
-    {
-        std::string stotal_credit = ExtractXML(vv, "<total_credit>", "</total_credit>");
-        std::string sexpavg_time = ExtractXML(vv, "<expavg_time>", "</expavg_time>");
-        std::string sexpavg_credit = ExtractXML(vv, "<expavg_credit>", "</expavg_credit>");
-        std::string scpid = ExtractXML(vv, "<cpid>", "</cpid>");
-
-        xmlout.nlappend("<user>");
-        xmlout.xmlappend("total_credit", stotal_credit);
-        xmlout.xmlappend("expavg_time", sexpavg_time);
-        xmlout.xmlappend("expavg_credit", sexpavg_credit);
-        xmlout.xmlappend("cpid", scpid);
-        xmlout.nlappend("</user>");
-    }
-
-    xmlout.nlappend("</users>");
-
-    stream << xmlout.value();
-
-    _log(INFO, "ProcessProjectRacFileByCPID", "Finished processing team rac data; stripping complete");
```

Comment:
This line needs to be re-added.

Reply:
Actually you combined the processing, so I don't think we need after all...